clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1511,6 +1511,28 @@ default DescribeTransactionsResult describeTransactions(Collection<String> trans
      */
     DescribeTransactionsResult describeTransactions(Collection<String> transactionalIds, DescribeTransactionsOptions options);
 
+    /**
+     * Forcefully abort a transaction which is open on a topic partition. See
+     * {@link #abortTransaction(AbortTransactionSpec, AbortTransactionOptions)} for more details.
+     *
+     * @param spec The transaction specification including topic partition and producer details
+     * @return The result
+     */
+    default AbortTransactionResult abortTransaction(AbortTransactionSpec spec) {
+        return abortTransaction(spec, new AbortTransactionOptions());
+    }
+
+    /**
+     * Forcefully abort a transaction which is open on a topic partition. This will
+     * send a `WriteTxnMarkers` request to the partition leader in order to abort the
+     * transaction. This requires administrative privileges.
+     *
+     * @param spec The transaction specification including topic partition and producer details
+     * @param options Options to control the method behavior (including filters)
+     * @return The result
+     */
+    AbortTransactionResult abortTransaction(AbortTransactionSpec spec, AbortTransactionOptions options);
```

Comment:
@hachikuji @dajac Sorry for waking up this PR, but I have a question: why it can't accept multiples `AbortTransactionSpec`? I did not find the answer from KIP-664 and related discussion. I know this method is used rarely. Hence, users don't hate kafka even if they need to call `abortTransaction` many times. However, I wonder whether any context I failed to catch :)